### ISSUE TITLE
Control Vocabulary & Tables

### ISSUE THREAD

**Issue by dnadeau4** [PCMDI/cmor#80]
- [x] Change "1x1"to "1x1 degree" in CV.json file
- [ ] Add  standard_name ocean_s_coordinate_g1 [Check]
- [ ] Add  standard_name ocean_s_coordinate_g2 [Check]
- [x] Split the axis info from the all CMIP6 tables into a new file.  (CMIP6_Coordinate.json)
- [x] Add all formula_vars to new axis CMIP6 file. (CMIP6_Formula_terms.json)
- [x] add sub_experiment verify SYYYY and create a validation list [S1960 --- S2020]  
- [x] Add sub_experiment (title) to each of these keys.
  - s1960: "initialized near end of year 1960"
- [x] Check file_template and directory_template for consistency with definition. 
- [x] There's a single case where the parent experiment is not strictly determined by the experiment_id: it can either be "none" or "dcppA-assim", and only the data provider can specify the correct one.

**Comment by dnadeau4** [comment]
@durack1 could you check that list make sure I did not miss anything!

**Comment by durack1** [comment]
@dnadeau4 you can delete the following bullet point (this is now included in the [CMIP6_experiment_id.json](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_experiment_id.json) file:
```
- recreate experiments from Excel spreadsheet with parent attributes
```
We are also waiting on MIP chairs to sort out the `sub_experiment` start and end dates for each experiment, so that can't be finalized - but the format `sYYYY` is correct as far as I know..

I'm not sure what other things were needed - didn't you take notes about this? We can chat tomorrow

**Comment by dnadeau4** [comment]
@durack1 these are my notes! :smile:

**Comment by durack1** [comment]
@dnadeau4 let's sit down at a screen and discuss this tomorrow

**Comment by dnadeau4** [comment]
@durack1 @taylor13 Is this still relevant?

**Comment by durack1** [comment]
@dnadeau4 I think not, as we now have the `CMIP6_CVs` repo and all the curated info there, this is no longer relevant I believe..

**Comment by dnadeau4** [comment]
@Taylor13 when you have sub_experiment settle I will implement it.  I am moving the milestone to 3.2.3

**Comment by dnadeau4** [comment]
@durack1  Is this still needed?
- [ ] Add  standard_name ocean_s_coordinate_g1 [Check]
- [ ] Add  standard_name ocean_s_coordinate_g2 [Check]

**Comment by durack1** [comment]
@dnadeau4 I believe this is now redundant with [WCRP-CMIP/CMIP6_CVs](https://github.com/WCRP-CMIP/CMIP6_CVs) so it's good to close
